dronekit/__init__.py

```diff
@@ -2658,8 +2658,7 @@ def thr_min_callback(self, attr_name, value):
         :param args: The callback to invoke when a change in the parameter is detected.
 
         """
-        attr_name = attr_name.upper()
-        return super(Parameters, self).add_attribute_listener(attr_name, *args, **kwargs)
+        return super(Parameters, self).add_attribute_listener(attr_name.upper(), *args, **kwargs)
```

Comment:
Mmmm.... this doesn't look like python3 support to me.

Reply:
Uhm, I think I changed this to print the parameter name while debugging and I forgot to change it back. Should be reverted (will do it).